plugins/woocommerce/composer.json

```diff
@@ -17,13 +17,13 @@
 		"php": ">=7.4",
 		"automattic/jetpack-autoloader": "2.11.18",
 		"automattic/jetpack-config": "1.15.2",
-		"automattic/jetpack-connection": "1.51.7",
+		"automattic/jetpack-connection": "^1.57",
```

Comment:
This change is added to fix the Jetpack dependency error. 
Slack: p1695840907172979/1695821353.214559-slack-C05EVAXCX7U